device-types/QNAP/TS-1685.yml

```diff
@@ -0,0 +1,38 @@
+---
+manufacturer: QNAP
+model: TS-1685-D1521-16G-550W
```

Comment:
The model name of this looks to be `TS-1658`. The part number is `TS-1685-D1521-16G-550W`

Please adjust accordingly

Reply:
@detvan besides the file name, also please modify the model definition to `TS-1658`